lang/fpc/Portfile

```
@@ -220,12 +220,15 @@ subport "${name}-cross-x86-64-win64" {
     }
 }
 
-if {${subport} eq "${name}"} {
+if {${subport} == "${name}"} {
     revision            1
 
     installs_libs       yes
 
-    extract.only-append ${pp}
+    # extract bzip2 archive including on systems (<=10.8) where gzip can't do it.
+    post-extract {
+       system -W ${workpath} "tar -xjf ${distpath}/${pp}"
```

Fix alignment and escape shell argument:
```suggestion
        system -W ${workpath} "tar -xjf [shellescape ${distpath}/${pp}]"
```

---

What does the escape shell argument do? I could not really find any infos about it.

---

The `shellescape` procedure escapes characters that are special to the shell so that the command will still work even if the variable value(s) contain special characters.